build-aux/Jenkinsfile.pr

```diff
@@ -247,7 +247,7 @@ pipeline {
         axes {
           axis {
             name 'ERLANG_VERSION'
-            values '25.3.2.20', '26.2.5.11', '27.3.3'
+            values '26.2.5.11', '27.3.3', '28.0'
```

Comment:
Note: we don't have a 28 runner yet. We are waiting on elixir support there

Reply:
Ok, let’s wait for it to land and then we can merge this PR.